### PR TITLE
PHC-4075 - Add @react-navigation/material-bottom-tabs

### PR DESCRIPTION
**Pull request by namick**
## Changes

  -  Add `@react-navigation/material-bottom-tabs` which requires and uses Paper's [Bottom Navigation](https://callstack.github.io/react-native-paper/docs/components/BottomNavigation/)

## Screenshots

https://user-images.githubusercontent.com/220893/225780252-5b15244d-4aba-407c-b6c2-cb60d3b615e9.mov

### CODE REVIEW

File: src/navigators/TabNavigator.tsx [hunk 1]
```diff
@@ -11,22 +11,34 @@ export type TabParamList = {
...
       <Tab.Screen
         name="HomeTab"
         component={HomeStack}
-        options={{ tabBarLabel: t('tabs-home', 'Home'), headerShown: false }}
+        options={{
+          tabBarLabel: t('tabs-home', 'Home'),
+          tabBarIcon: 'home',
+        }}
+      />
+      <Tab.Screen
```
Comment (bruce-glazier) [inline src/navigators/TabNavigator.tsx:27]: Love the way it looks but I think we should probably hide the notification tab until we have content for it.

File: package.json [hunk 2]
```diff
@@ -27,6 +27,7 @@
     "@react-native-community/eslint-config": "^3.2.0",
     "@react-native-community/netinfo": "^9.3.7",
     "@react-navigation/bottom-tabs": "^6.5.5",
+    "@react-navigation/material-bottom-tabs": "^6.2.15",
```
Comment (markdlv) [inline package.json:30]: Can the `@react-navigation/bottom-tabs` dep above this be removed, or is it still needed?

Comment (namick) [reply package.json:29]: Yep, removed.

File: src/navigators/TabNavigator.tsx (hunk 1 above)
Comment (namick) [reply src/navigators/TabNavigator.tsx:26]: Roger that.

**Pull request merged**